installer/templates/phx_umbrella/apps/app_name_web/lib/templates/page/index.html.eex

```diff
@@ -1,5 +1,36 @@
-<div id="container" class="row marketing">
+<div class="jumbotron">
+  <h2><%%= gettext "Welcome to %{name}", name: "Phoenix!" %></h2>
```

Comment:
Maybe leave the exclamation mark inside the text to translate (Spanish for instance does special things with that)?